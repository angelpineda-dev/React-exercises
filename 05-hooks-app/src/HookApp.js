import CounterApp from "./components/01-useState/CounterApp";

const HookApp = () => {
  return (
    <div>
      <h1>Hooks</h1>
      <br />
      <CounterApp />
    </div>
  );
};

export default HookApp;
