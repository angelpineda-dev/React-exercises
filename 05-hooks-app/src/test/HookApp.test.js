import HookApp from "../HookApp";
import { shallow } from "enzyme";

describe("Pruebas en <HookApp/>", () => {
  const wrapper = shallow(<HookApp />);

  test("Debe mostrar el componente correctamente", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
