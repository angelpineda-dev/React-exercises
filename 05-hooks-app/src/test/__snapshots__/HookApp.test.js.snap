// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp/> Debe mostrar el componente correctamente 1`] = `
<div>
  <h1>
    Hooks
  </h1>
  <hr />
  <MainApp />
  <hr />
  <Padre />
  <hr />
  <CallbackHook />
  <hr />
  <MemoHook />
  <hr />
  <Memorize />
  <hr />
  <Layout />
  <hr />
  <RealExampleRef />
  <hr />
  <FocusScreen />
  <hr />
  <MultipleCustomHooks />
  <hr />
  <FormWithCustomHook />
  <hr />
  <SimpleForm />
  <hr />
  <CounterWithCustomHook />
  <hr />
  <CounterApp />
</div>
`;
